index.html

```diff
@@ -48,7 +52,7 @@
       <section id="entries" class="tab-zone">
       </section>
       <section id="about" class="tab-zone">
-        <h2>About Lex</h2>
+        <!-- <h2>About Lex</h2>
```

Comment:
this comment can be deleted because about-handlebars-template its populating that.